src/shadowbox/server/run_action.sh

```diff
@@ -25,9 +22,15 @@ export SB_PUBLIC_IP="${SB_PUBLIC_IP:-$(curl https://ipinfo.io/ip)}"
 export SB_API_PREFIX=TestApiPrefix
 export SB_METRICS_URL=https://metrics-test.uproxy.org
 export SB_STATE_DIR=/tmp/outline
+mkdir -p ${SB_STATE_DIR}
+export SB_BIN_DIR=${SB_BIN_DIR:-$(mktemp -d)}
```

Comment:
I'm afraid we are ending up with many directories. I believe we can simplify that and  define a SB_ROOT_DIR instead, that contains bin/ and persisted-state/. That's then the only variable you need to set (all the rest are default subdirs)

We should then strive to keep everything within that directory (including the IP->location table)

We'll need to change `getPersistentFilename()`.

Reply:
Done

Reply:
Oh whoops didn't move the geolocation db

Reply:
Now this is ready to go